src/platform/silabs/ConfigurationManagerImpl.cpp

```diff
@@ -149,6 +149,14 @@ CHIP_ERROR ConfigurationManagerImpl::GetBootReason(uint32_t & bootReason)
     matterBootCause = BootReasonType::kUnspecified;
 #endif
 
+#if !SLI_SI91X_MCU_INTERFACE
+    BootloaderResetCause_t testBootReason = bootloader_getResetReason();
```

Comment:
Should this still be here

Reply:
Yes if I took the time to add the proper include and sources... so No. Will remove.